lib/state/appState.js

```diff
@@ -68,6 +68,7 @@ appState.getInitial = function() {
 
   return {
     dropMenu: false,
+    howToUpdateKBLink: 'https://tidepool-project.helpscoutdocs.com/article/6-how-to-install-or-upgrade-the-tidepool-uploader-gen',
```

Comment:
For future reference, the new Trello card:

https://trello.com/c/PHhrxxYn/97-2-uploader-create-redirect-url-endpoint-for-how-to-update-kb-article-in-uploader <https://trello.com/c/PHhrxxYn/97-2-uploader-create-redirect-url-endpoint-for-how-to-update-kb-article-in-uploader>


> On Aug 13, 2015, at 10:49 AM, Jana Beck <notifications@github.com> wrote:
> 
> In lib/state/appState.js <https://github.com/tidepool-org/chrome-uploader/pull/139#discussion_r37003468>:
> 
> > @@ -68,6 +68,7 @@ appState.getInitial = function() {
> >  
> >    return {
> >      dropMenu: false,
> > +    howToUpdateKBLink: 'https://tidepool-project.helpscoutdocs.com/article/6-how-to-install-or-upgrade-the-tidepool-uploader-gen',
> Sure, can you create that card @darinkrauss <https://github.com/darinkrauss>?
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/tidepool-org/chrome-uploader/pull/139/files#r37003468>.
>